content/docs/atom-guide/setup-atom-projects.md

```diff
@@ -1,111 +1,66 @@
 
+Before setting up a new O3DE project or Atom Sample Viewer, you must install O3DE on your computer. Follow the instructions in [Setting Up O3DE](/docs/welcome-guide/setup) of the Get Started Guide.
 
-## Setting up the Atom projects
 
-This tutorial uses the following project name and directories in the examples:
+## O3DE Project
 
-- O3DE engine directory: `C:\o3de`
-- Atom Test project directory: `C:\AtomTest`
-- Atom Sample Viewer project directory: `C:\AtomSampleViewer`
-- Package directory (created earlier during [setup](/docs/welcome-guide/setup/setup-from-github/)): `C:\o3de-packages`
+Creating an O3DE project provides you the full suite of tools, including **O3DE Editor**, **Material Editor**, and a collection of Gems essential to building a simulation or game. This is a great way to test out Atom's features among other objects and mechanics in a level. 
 
-## Clone project repos
+![Default level prefab](/images/learning-guide/tutorials/environments/create-a-level-C.png)
 
-To get a local copy of the Atom projects, you must clone their repository from GitHub. For best practice, clone the repo into a directory outside of the O3DE repository. 
+### Setup 
 
-Atom Test
-: 
-    ```cmd
-    git clone https://github.com/o3de/o3de-atomtest.git C:\AtomTest
-    ```
+1. Follow the instructions in[Creating a project](/docs/welcome-guide/create/) of the Get Started guide. Atom is built into O3DE via the **Atom Gem**, a core Gem that's enabled in your project by default.
 
-Atom Sample Viewer
-: 
-    ```cmd    
-    git clone https://github.com/o3de/o3de-atom-sampleviewer.git C:\AtomSampleViewer
-    ```
+2. After you set up your project, check out the following pages to explore Atom's features via the O3DE Editor:
+   - [Create a Level](/docs/learning-guide/tutorials/environments/create-a-level/): An introduction to your first level. Whenever you create a level in your project, the default environment consists of many Atom components. 
+   - [Component Reference - Atom](/docs/user-guide/components/reference/#atom): The full list of Atom components provided by the Atom Gem. 
 
-## Update the local project repos
 
-The community is continuing to update the Atom project repositories. This step ensures your local copy has the latest updates from the Atom repositories. 
+## Atom Sample Viewer
 
-In a command line window, navigate to the directory of the Atom repository and pull the latest changes from `origin/main` into your branch. 
+Atom Sample Viewer is a good place to start if you're interested in evaluating Atom's features in a controlled environment. It's a standalone application, meaning you don't need to open it with O3DE Editor. Rather, it showcases samples that demonstrate each feature individually and with parameters you can tweak and configure. 
 
-Atom Test
-: 
-    ```cmd
-    cd C:\AtomTest
-    git pull origin main
-    ```
+Check out the [`o3de-atom-sampleviewer`](https://github.com/o3de/o3de-atom-sampleviewer) repository in GitHub.
```

Comment:
```suggestion
To use the Atom Sample Viewer, get a local copy from the [`o3de-atom-sampleviewer`](https://github.com/o3de/o3de-atom-sampleviewer) repository in GitHub.
```
Added some additional context for the GitHub link.

Reply:
I decided to just remove this line because I didn't want to give the user an action item at this point in the doc. Especially since immediately after are instructions to get the repository.